less/ng-obiba-mica.less

```diff
@@ -389,7 +389,17 @@ span.search-help {
 }
 
 .obiba_mica_lists {
-  .read-more-btn{
-    margin-bottom: 15px !important;
+  .caption-count{
+    position: absolute;
+    bottom: 0;
+  }
+  a.label{
+    font-size: inherit;
+    color: @link-color;
+    padding: 0px;
+  }
+  a.label:hover{
+    color: @link-hover-color;
+    text-decoration: underline;
```

Comment:
I really do not understand what you are doing, but I have a bad feeling on the result...